src/components/Arpaname/arpaname.tsx

```diff
@@ -0,0 +1,108 @@
+import { Button, LoadingOverlay, Stack, TextInput, Title } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { useCallback, useState } from "react";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { UserGuide } from "../UserGuide/UserGuide";
+import { SaveOutputToTextFile } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+
+const title = "Arpaname Tool";
+const description_userguide =
+    "Arpaname translates IP addresses (IPv4 and IPv6) to the corresponding IN-ADDR.ARPA or IP6.ARPA names.";
+
+interface FormValuesType {
+    ipAddress: string;
+}
+
+const ArpanameTool = () => {
+    const [loading, setLoading] = useState(false);
```

Comment:
- [ ] Add component constant section and comment
- [ ] Add component state variable section and comment